lib/features/profiles/widgets/profile_item.dart

```diff
@@ -0,0 +1,49 @@
+// ignore_for_file: prefer_const_constructors
+
+import 'package:flutter/material.dart';
+
+class ProfileItem extends StatelessWidget {
+  const ProfileItem({
+    Key? key,
+    required this.name,
+    required this.imageUrl,
+  }) : super(key: key);
+
+  final String name;
+  final String imageUrl;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      alignment: Alignment.center,
+      child: Stack(
+        children: [
+          Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              ClipRRect(
+                borderRadius: BorderRadius.circular(15),
+                child: Image.network(
```

Comment:
In case you want to save the profile image at some point you can use the [CachecImageNetwork](https://pub.dev/packages/cached_network_image)

Reply:
I did not know about it, thanks!